### PR TITLE
rbd-nbd: support disabling auto-exclusive lock transition logic

### PR DESCRIPTION
**Pull request by trociny**
Fixes: http://tracker.ceph.com/issues/17488
Signed-off-by: Mykola Golub <mgolub@mirantis.com>

### CODE REVIEW

File: src/tools/rbd_nbd/rbd-nbd.cc [hunk 1]
```diff
@@ -64,13 +64,15 @@ static void usage()
             << "  --device <device path>                    Specify nbd device path\n"
             << "  --read-only                               Map readonly\n"
             << "  --nbds_max <limit>                        Override for module param\n"
+            << "  --single-client                           Forbid other clients write\n"
```
Comment (dillaman) [inline src/tools/rbd_nbd/rbd-nbd.cc]: Minor: how about just "--exclusive" or "--exclusive-client"? Also, can you add the argument to the "rbd nbd map" CLI as well?

**Comment by dillaman** [comment]
... also need to update the rbd CLI help.t cram test case

**Comment by trociny** [comment]
@dillaman Thanks, updated.

**Comment by dillaman** [comment]
lgtm

**Pull request merged**